openzaak-alfresco-share/src/main/resources/alfresco/web-extension/messages/openzaak-alfresco-share.properties

```
@@ -0,0 +1,95 @@
+form.zaak.set.zaaktypeinformatie.zaaktypeurl.label=Zaaktype url
+
+# Case messages
+form.advancedsearch.zaak.label=Zaak zoeken
+form.advancedsearch.zaak.description=Zoeken naar Openzaak zaken
+
+form.zaak.set.openzaak.label=Zaak informatie
+form.zaak.set.openzaak.zaakurl.label=Zaak url
+form.zaak.set.openzaak.zaaktypeurl.label=Zaaktype url
+form.zaak.set.openzaak.zaakidentificatie.label=Zaak identificatie
+form.zaak.set.openzaak.zaakbronorganisatie.label=Zaak bronorganisatie
+
+# Case document messages
+form.advancedsearch.document.label=Document zoeken
+form.advancedsearch.document.description=Zoeken naar Openzaak documenten
+
+form.document.set.basisinformatie.label=Basis informatie
+form.document.set.basisinformatie.uuid.label=Document UUID
+form.document.set.basisinformatie.bestandsnaam.label=Bestandsnaam
+form.document.set.basisinformatie.identificatie.label=Identificatie
+form.document.set.basisinformatie.titel.label=Titel
+form.document.set.basisinformatie.beschrijving.label=Beschrijving
+form.document.set.basisinformatie.informatieobjecttype.label=Informatie object type
+form.document.set.basisinformatie.creatiedatum.label=Creatie datum
+form.document.set.basisinformatie.auteur.label=Auteur
+form.document.set.basisinformatie.taal.label=Taal
+
+form.document.set.documentinformatie.label=Document informatie
+form.document.set.documentinformatie.bronorganisatie.label=Bron organisatie
+form.document.set.documentinformatie.link.label=Link
+form.document.set.documentinformatie.verwijderd.label=Verwijderd
+form.document.set.documentinformatie.formaat.label=Formaat
+form.document.set.documentinformatie.versie.label=Versie
+form.document.set.documentinformatie.lock.label=Lock
+form.document.set.documentinformatie.kopievan.label=Kopie van
+
+form.document.set.progressieinformatie.label=Progressie
+form.document.set.progressieinformatie.verzenddatum.label=Verzenddatum
+form.document.set.progressieinformatie.ontvangstdatum.label=Ontvangstdatum
+form.document.set.progressieinformatie.beginregistratie.label=Begin registratie
+form.document.set.progressieinformatie.status.label=Status
+
+form.document.set.ondertekeningsinformatie.label=Ondertekening
+form.document.set.ondertekeningsinformatie.ondertekeningdatum.label=Datum ondertekening
+form.document.set.ondertekeningsinformatie.ondertekeningsoort.label=Soort ondertekening
+
+form.document.set.integriteitsinformatie.label=Integriteit
+form.document.set.integriteitsinformatie.integriteitwaarde.label=Integriteit waarde
+form.document.set.integriteitsinformatie.integriteitdatum.label=Integriteitdatum
+form.document.set.integriteitsinformatie.integriteitalgoritme.label=Integriteitalgoritme
+
+form.document.set.gebruiksinformatie.label=Gebruik document
+form.document.set.gebruiksinformatie.vertrouwelijkaanduiding.label=Vertrouwelijkheids aanduiding
+form.document.set.gebruiksinformatie.indicatiegebruiksrecht.label=Indicatie gebruiksrecht
+
+# OIO messages
+form.advancedsearch.oio.label=OIO zoeken
```

```suggestion
form.advancedsearch.oio.label=ObjectInformatieObject zoeken
```
& volgende entries kunnen dan ook volledige woord gebruiken